packages/admin/src/common/toolbar/backbutton/ToolbarBackButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { IconButton } from "@material-ui/core";
+import * as React from "react";
+
+import { useStackApi } from "../../../stack";
+import { ToolbarItem } from "../item/ToolbarItem";
+import { useStyles, useToolbarBackButtonThemeProps } from "./ToolbarBackButton.styles";
+
+interface ToolbarBackButtonProps {
+    component?: (goBack?: () => void) => React.ReactNode;
+}
+export function ToolbarBackButton({ component }: ToolbarBackButtonProps) {
```

Comment:
A relatively simple component `ToolbarBackButton`, takes another component as prop to override itself? Is this prop useful? We could also omit it for the sake of simplicity?

Reply:
true story, good point 👍 i will remove that and update the custom back button story.

Reply:
removed custom component rendering of `toolbarbackbutton`